server/models/study.py

```diff
@@ -176,6 +186,54 @@ def getImages(self, study):
             'meta.studyId': study['_id']}).distinct('meta.imageId')
         return Image().find({'_id': {'$in': imageIds}})
 
+    def addWaitingUser(self, study, user):
+        """
+        Add user to list of users waiting to participate in the study.
+        """
+        waitingUsers = study['meta'].get('waitingUsers', [])
```

Comment:
We can assume this always exists (here, and in other methods). I'll run a migration to add the field.